qiskit/providers/aer/backends/aerbackend.py

```diff
@@ -110,42 +110,23 @@ def __init__(self,
     @deprecate_arguments({'qobj': 'circuits'})
     def run(self,
             circuits,
-            backend_options=None,  # DEPRECATED
             validate=False,
             **run_options):
         """Run a qobj on the backend.
 
         Args:
             circuits (QuantumCircuit or list): The QuantumCircuit (or list
                 of QuantumCircuit objects) to run
-            backend_options (dict or None): DEPRECATED dictionary of backend options
-                                            for the execution (default: None).
             validate (bool): validate the Qobj before running (default: False).
```

Comment:
is this used anywhere now, the deprecation warning about validate in `_run_job` was removed below I didn't think it was used anymore.

Reply:
We opted to turn validation off by default, but I don't think we have deprecated it. We could do so though since I don't think its used.